app/search/query/tests/test_base.py

```diff
@@ -0,0 +1,81 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from nose.tools import eq_, ok_
+
+from app.search.query.base import BaseQueryEngine
+from app.tests.memcached import mock_memcached
+
+
+QUERY = 'loveland'
+MOCK_RESULTS = [
+    {
+        'abstract': 'Welcome to <b>Loveland</b> Ski Area Near to the...',
+        'title': '<b>Loveland</b> Ski Area :: Colorado Ski Snowboarding...',
+        'url': 'http://skiloveland.com/'
+    },
+    {
+        'abstract': 'Loveland Ski Area is a ski area in the western United...',
+        'title': 'Loveland Ski Area - Wikipedia, the free encyclopedia',
+        'url': 'https://en.wikipedia.org/wiki/Loveland_Ski_Area'
+    }
+]
+MOCK_RESULT_SANITIZED = {
+    'abstract': 'Welcome to Loveland Ski Area Near to the...',
+    'title': 'Loveland Ski Area :: Colorado Ski Snowboarding...',
+    'url': 'http://skiloveland.com/'
+}
+
+
+class TestBaseQueryEngine(TestCase):
+    def setUp(self):
+        self.instance = BaseQueryEngine('')
+
+    def test_fetch(self):
+        with self.assertRaises(NotImplementedError):
+            self.instance.fetch(QUERY)
+
+    def test_sanitize_response(self):
+        eq_(self.instance.sanitize_response(MOCK_RESULTS), MOCK_RESULTS)
+
+    def test_get_best_result(self):
+        eq_(self.instance.get_best_result(MOCK_RESULTS), MOCK_RESULTS[0])
+
+    @patch('app.search.query.base.BaseQueryEngine.get_result_abstract')
+    @patch('app.search.query.base.BaseQueryEngine.get_result_title')
+    @patch('app.search.query.base.BaseQueryEngine.get_result_url')
+    def test_sanitize_result(self, *args):
+        result = self.instance.sanitize_result(MOCK_RESULTS[0])
+        ok_(all(k in result.keys() for k in ['abstract', 'title', 'url']))
+        ok_(all([mock.call_count == 1 for mock in args]))
+
+    def test_strip_html(self):
```

Comment:
Same as the previous comment. I tend not to duplicate tests that libraries explicitly test for, but I'd be happy to do so if you deem it worthy.